src/status_im/multiaccounts/login/core.cljs

```diff
@@ -479,9 +479,13 @@
 (defn redirect-to-root
   "Decides which root should be initialised depending on user and app state"
   [db]
-  (if (get db :tos/accepted?)
-    (re-frame/dispatch [:init-root :shell-stack])
-    (re-frame/dispatch [:init-root :tos])))
+  (let [tos-accepted?            (get db :tos/accepted?)
+        completed-local-pairing? (get db :local-pairing/completed-pairing?)]
+    (if tos-accepted?
+      (if completed-local-pairing?
```

Comment:
it could be just `(:local-pairing/completed-pairing? db)` no need in extra let

Reply:
I added `let` for better readability. I will remove `get` and replace it as per the above suggestion.